signingvalidation.proj

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's kind of weird that this folder is named `signingvalidation.proj`.  Can it just be renamed to `signingvalidation`?

Or maybe moving this new file and the `sign.proj` file into a `signing` folder?